src/pages/Lending/Pool/components/PoolInfo.tsx

```diff
@@ -60,6 +75,58 @@ export const PoolInfo = ({ poolAssetId }: PoolInfoProps) => {
     [poolData?.totalBorrowers],
   )
 
+  const renderVaultCap = useMemo(() => {
+    if (!poolData || !asset) return null
+
+    const isHardCapReached = bnOrZero(poolData.tvl).eq(poolData.maxSupplyFiat)
+    const currentCapFillPercentage = bnOrZero(poolData.tvl)
+      .div(bnOrZero(poolData.maxSupplyFiat))
+      .times(100)
+      .toNumber()
+
+    return (
+      <Flex flexWrap='wrap' direction='column' gap={4}>
+        <Flex justifyContent='space-between' alignItems='center'>
+          <HelperTooltip label={translate('defi.modals.saversVaults.vaultCapTooltip')}>
+            <Text fontWeight='medium' translation='defi.modals.saversVaults.vaultCap' />
+          </HelperTooltip>
+          <Flex gap={1}>
+            <Amount.Fiat value={poolData?.tvl} />
+            <Amount.Fiat value={poolData?.maxSupplyFiat} prefix='/' color='text.subtle' />
+          </Flex>
+        </Flex>
+        {isHardCapReached || bnOrZero(currentCapFillPercentage).eq(100) ? (
+          <Alert status='warning' flexDir='column' bg={alertBg} py={4}>
+            <AlertIcon />
+            <AlertTitle>{translate('defi.modals.saversVaults.haltedTitle')}</AlertTitle>
+            <>
+              <AlertDescription>
+                {translate('defi.modals.saversVaults.haltedDescription')}
+              </AlertDescription>
+              <Button
+                as={Link}
+                href={`https://twitter.com/intent/tweet?text=Hey%20%40THORChain%20%23raisethecaps%20already%20so%20I%20can%20lend%20%23${asset.symbol}%20on%20%40ShapeShift`}
+                isExternal
+                mt={4}
+                colorScheme='twitter'
+                rightIcon={faTwitterIcon}
+              >
+                @THORChain
+              </Button>
+            </>
+          </Alert>
+        ) : (
+          <Progress
+            value={currentCapFillPercentage}
+            size='sm'
+            borderRadius='md'
+            colorScheme={bnOrZero(currentCapFillPercentage).lt(100) ? 'green' : 'red'}
+          />
+        )}
+      </Flex>
+    )
+  }, [alertBg, asset, poolData, translate])
+
```

Comment:
nitpicky but is there a reason this guy isnt a separate component?

Reply:
Not really, I guess just not generic/reused enough to be moved to its own component?